.github/workflows/create-summary-pr.yml

```
@@ -10,8 +10,16 @@ jobs:
     name: Create Summary Update PR
     runs-on: ubuntu-latest
     steps:
-      - uses: actions/checkout@v2
+      - name: Checkout Repository
```

```suggestion
      - name: Checkout repository
```
Let's make naming consistent. Either use title case or sentence case.